src/azure-cli/azure/cli/command_modules/profile/breaking_change.py

```diff
@@ -0,0 +1,11 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+from azure.cli.core.breaking_change import register_conditional_breaking_change
+
+register_conditional_breaking_change(tag='ManagedIdentityUsernameBreakingChange', breaking_change=(
+    'login',
+    'Passing the managed identity ID with --username is deprecated and will be removed in a future release. '
```

Comment:
If we plan to deprecate this behavior in next breaking change window, could we specify the target release as an explicit version, like `2.73.0` for example?